Bug OCPBUGS-2886: Add note about unnecessary load-balancer setting on CP (#52293)
openshift/openshift-docs: modules/installation-osp-setting-cloud-provider-options.adoc

. Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+

> Configuring Octavia for load balancing is a common case for installations that do not use Kuryr. For example:

Reviewer: @MaysaMacedo Actually, could one of these work better?

"installations" -> "clusters"
"installations" -> "deployments" 

After that, I think this just needs a QE ack.

Author: clusters I think is better. Let me update it

====
You must set the value of the `create-monitor` property to `True` if you use services that have the value of the `.spec.endpointTrafficPolicy` property set to `Local`. The OVN Octavia provider in {rh-openstack} 16.1 and 16.2 does not support health monitors. Therefore, services that have `ETP` parameter values set to `Local` might not respond when the `lb-provider` value is set to `"ovn"`.
====
+
[IMPORTANT]
====

> For installations with Kuryr, the Services are handled by Kuryr, consequently there is no need to configure octavia load balancing in the cloud provider.

Reviewer: ```suggestion
For installations that use Kuryr, Kuryr handles relevant services. There is no need to configure Octavia load balancing in the cloud provider.
```
